Src/Plugins/Security/SEC101.SecurePlaintextSecrets.json

```diff
@@ -132,9 +132,9 @@
         },
         {
           "Id": "SEC101/015",
-          "Name": "DoNotExposePlaintextSecrets/AkamaiKey",
-          "ContentsRegex": "$SEC101/015.AkamaiKey",
-          "MessageArguments": { "secretKind": "Akamai key" }
+          "Name": "DoNotExposePlaintextSecrets/AkamaiCredentials",
```

Comment:
>AkamaiCredentials [](start = 47, length = 17)

This secret kind requires multiple components, and so by convention the proper name is 'credentials'.